docs/content/platform/support-guide.md

```
@@ -0,0 +1,180 @@
+
+Do: Explicitly state the limits and provide a clear alternative.
+
+Script: "I’d recommend not automating that [specific task], it’s not a great fit for automation right now because [reason]. However, [Task B] is ideal and could save you a lot of time. Let's look at that."
+
+## 🛠️ Section 2: Installation & Technical Troubleshooting
+
+### Q: How do I install AutoGPT locally?
+
+For most users, recommend the Auto Setup Script:
+
+MacOS/Linux: curl -fsSL https://setup.agpt.co/install.sh -o install.sh && bash install.sh
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)